### PR TITLE
Allow loading sps and pps values before sending frames

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

**Comment by ConnorRigby** [comment]
This fixes #6

File: lib/membrane_element_ffmpeg_h264/decoder.ex [hunk 1]
```diff
@@ -105,4 +110,12 @@ defmodule Membrane.Element.FFmpeg.H264.Decoder do
   end
 
   defp get_caps_if_needed(_, _, _), do: {:ok, []}
+
+  defp maybe_decode(nil, _decoder_ref), do: :ok
+
+  defp maybe_decode(data, decoder_ref) do
+    # ignore return of this function as ffpmeg treats it as an error.
```
Comment (mat-hek) [inline lib/membrane_element_ffmpeg_h264/decoder.ex]: Maybe waiting for the first buffer and prepending the parameter sets when it incomes would help avoiding that error?

Comment (ConnorRigby) [reply lib/membrane_element_ffmpeg_h264/decoder.ex]: You're correct. I just tested that out, and it removes the error. Will update the PR

Comment (ConnorRigby) [reply lib/membrane_element_ffmpeg_h264/decoder.ex]: @mat-hek i've updated the PR. Let me know what you think.

**Pull request merged**